app/Http/Controllers/Api/WaitingRoomsController.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Gladiator\Http\Controllers\Api;
+
+use Illuminate\Http\Request;
+use Gladiator\Http\Controllers\Controller;
+use Gladiator\Models\WaitingRoom;
+
+class WaitingRoomsController extends Controller
+{
+
+    /**
+     * Get a waiting room for the specified campaign ID
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function getWaitingRoom(Request $request)
+    {
+        $this->validate($request, [
+            'campaign_id' => 'required|integer'
```

Comment:
Im worried about using a WaitingRoomRequest here because it would be different than a normal request

(in this case, only campaign id is required, but in all other normal waiting room requests everything should be required)